.gitbook/1-users/3-data-exchange/2-provide-data.md

```diff
@@ -1 +1,74 @@
-# Consume data
+# Provide data
+
+Data providers can provide their data that match the requirements of the data deal and in return earn a reward.
+
+The provider request verification of the data to oracle, and submits consent with oracle signature to the Panacea public blockchain.
+
+Since the data is encrypted throughout the whole process of data transmission, no one can get the original data.
+
+The transmission of data and the payment of rewards are managed atomically through the Panacea public blockchain, so consumers can get the data only when the reward payment is completed.
```

Comment:
I added why user should first request to oracle for data verification.
I thought that user might feel a gap between submitting a consent and request for data verification to oracle.

```suggestion
Data providers can provide their data that match the requirements of the data deal and in return earn a reward.

Since only the data verified by oracle can be provided to a deal, providers should first request data verification to oracle.

As a result of verification, oracle will issue a certificate, then provider can provide their data by submitting a consent with the certificate to Panacea.

In the whole process of data transmission, the data is encrypted so that no one can access the original data.

The transmission of data and the payment of rewards are managed atomically through the Panacea public blockchain, so consumers can get the data only when the reward payment is completed.
```